src/attributes/views/AttributeCreate/AttributeCreate.tsx

```diff
@@ -54,6 +57,39 @@ function areValuesEqual(
   return a.name === b.name;
 }
 
+function getSimpleAttributeData(
+  data: AttributePageFormData,
+  values: AttributeValueEditDialogFormData[]
+) {
+  return {
+    ...data,
+    metadata: undefined,
+    privateMetadata: undefined,
+    storefrontSearchPosition: parseInt(data.storefrontSearchPosition, 0),
+    values: values.map(value => ({
+      name: value.name
+    }))
+  };
+}
+
+function getFileAttributeData(
+  data: AttributePageFormData,
+  values: AttributeValueEditDialogFormData[]
+) {
+  return {
+    ...data,
+    availableInGrid: undefined,
+    filterableInDashboard: undefined,
+    filterableInStorefront: undefined,
+    metadata: undefined,
+    privateMetadata: undefined,
+    storefrontSearchPosition: parseInt(data.storefrontSearchPosition, 0),
```

Comment:
```suggestion
    storefrontSearchPosition: parseInt(data.storefrontSearchPosition, 10),
```
https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/parseInt

Yes, I know, there are few occurances in the code ith 0 existing. I'll remove them during eslint update.